### DISCUSSION TITLE
Some GitHub API showing not found

### DISCUSSION THREAD

**Discussion by liorgolan-yakim** [community/community#51758] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

I started to use GItHub API and I have problem that I just can't understand why it's not working.
I installed GitHub CLI on my computer, used gh auth login and entered a personal token I have created with the right permissions.

If I am running for example an API for listing users/teams/specific user/team, it's working. using this:

User:
gh api \
  -H "Accept: application/vnd.github+json" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  /users/USERNAME

Team:
h api \
  -H "Accept: application/vnd.github+json" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  /orgs/ORG/teams/TEAM_SLUG

Both are working with the CLI
But when I want to run APi of the repo environments, every APi give me NOT FOUND error.

gh api \
  -H "Accept: application/vnd.github+json" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  /repos/OWNER/REPO/environments

or this (For that command there is no gh, only curl for some reason):
curl -L \
  -X PUT \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <YOUR-TOKEN>"\
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/OWNER/REPO/environments/ENVIRONMENT_NAME \
  -d '{"wait_timer":30,"reviewers":[{"type":"User","id":1},{"type":"Team","id":1}],"deployment_branch_policy":{"protected_branches":false,"custom_branch_policies":true}}'

So first I tried to understand why I need Bearer if I authenticated, so I with specifying token and without (because I put it in the gh auth login phase). The first environment API don't have a bearer option.
But both not working...
I thought maybe I am wrong on the OWNER, but on getting a team it works and there is an OWNER there as well.

NOTE: Of course I changed the ORG OWNER and the other parameters names when I was running it

**Comment by devthejo** [comment]
same when repository is private
